Анимируем смены состояний интерактивных элементов

0snilcy opened the pull request:
---
:mortar_board: [Анимируем смены состояний интерактивных элементов](https://up.htmlacademy.ru/animation/1/user/42701/tasks/2)

:boom: https://htmlacademy-animation.github.io/42701-magic-vacation-1/1/

0snilcy committed:
1.13. Анимируем смены состояний интерактивных элементов

gkarev commented:
Приветствую! ОТличная работа, но можем сделать её чуточку лучше)

Необходимо сделать плавным увеличение кнопки "поделиться" сейчас работает рывком. также рекомендую не использовать для этого JS. Достаточно просто решается с `hover`
https://skr.sh/vBBg6Q39E0Q?a

Анимация блока "ОТГАДАЙ ПЕРВЫМ МЕСТО" Реализована через `animation` и ключевые кадры, что несколько избыточной для анимации, которая имеет линейный сценарий, то есть скрыт/показан, без повторений и тп. Рекомендую использовать `transition`, обязательно укажите transition-proprety, смещение делается через `transform`. Аналогично и для блоков на странице "История" (текст в слайдах).

**Не работает.**
> На экране «Игра» сделайте плавное появление текстового поля. Поле должно появляться справа налево.

жду правки)!

0snilcy committed:
Правки ревью
https://github.com/htmlacademy-animation/42701-magic-vacation-1/pull/1#issuecomment-978011500

0snilcy commented:
> 



> Приветствую! ОТличная работа, но можем сделать её чуточку лучше)
> 
> Необходимо сделать плавным увеличение кнопки "поделиться" сейчас работает рывком. также рекомендую не использовать для этого JS. Достаточно просто решается с `hover` https://skr.sh/vBBg6Q39E0Q?a
> 
> Анимация блока "ОТГАДАЙ ПЕРВЫМ МЕСТО" Реализована через `animation` и ключевые кадры, что несколько избыточной для анимации, которая имеет линейный сценарий, то есть скрыт/показан, без повторений и тп. Рекомендую использовать `transition`, обязательно укажите transition-proprety, смещение делается через `transform`. Аналогично и для блоков на странице "История" (текст в слайдах).
> 
> **Не работает.**
> 
> > На экране «Игра» сделайте плавное появление текстового поля. Поле должно появляться справа налево.
> 
> жду правки)!

Привет, спасибо.
- JS я не добавлял он там был изначально, переписал на hover.
- Переписал анимацию блоков текста на вкладке _История_ на transition.
- Как мне сделать анимацию на вкладке _Главная_ для блока _Отгадай первым место_ на transition, если там нет изменений. Блок отрисовывается какой есть и все. То есть transition там не отработает.
- Как не работает появление поля, если работает?

https://user-images.githubusercontent.com/11254600/143784191-207bc905-3a6e-4d7a-ab0e-29a00d0e74ea.mp4

Спасибо!

gkarev commented:
>Как не работает появление поля, если работает?

на экране > 2500px све плохо.

>Как мне сделать анимацию на вкладке Главная для блока Отгадай первым место на transition, если там нет изменений. Блок отрисовывается какой есть и все. То есть transition там не отработает.

В следующем таске, будет привязка к классу при загрузки страницы, а пока - да, можно оставить и так.

0snilcy committed:
Правки ревью
https://github.com/htmlacademy-animation/42701-magic-vacation-1/pull/1#issuecomment-981167212

0snilcy commented:
Все нашел и поправил! :smiley_cat:
